src/webview/services/playerService.ts

```diff
@@ -122,12 +125,27 @@ export default class PlayerService extends Service {
 
   // seekbar value is 0~100
   public onSeekbarInput(value: number) {
+    const resumeRequired: boolean = this._isPlaying;
```

Comment:
(You don't need to fix this in this time)

You may write types explicitly sometimes and not at other times, but in this project you can basically rely on type inference, so you don't need to write them.
(Probably you're not really deciding whether to write them or not because of some particular preference, but just writing them sometimes and not at other times, right?)

It's true that there are benefits to being explicit, such as faster compilation, but I don't think there's much benefit, especially in a trivial case like this one.

Reply:
Thank you for pointing that out. 
I sometimes write type definitions explicitly because I'm still becoming familiar with TypeScript. 
I understand the direction of this project now.